lib/measures/tbd/resources/psi.rb

```diff
@@ -188,133 +187,138 @@ def processTBDinputs(surfaces, edges, set, io_path, schema_path)
     raise "processTBDinputs: TBD edges class #{edges.class}?"
   end
 
-  # Validate TBD JSON schema.
-  raise "processTBDinputs: TBD schema file?" unless File.exist?(schema_path)
-  raise "processTBDinputs: Empty TBD schema file?" if File.zero?(schema_path)
-  schema_c = File.read(schema_path)
-  schema = JSON.parse(schema_c, symbolize_names: true)
-
-  if File.size?(io_path) # optional input file exists and is non-zero
+  if io_path && File.size?(io_path) # optional input file exists and is non-zero
     io_c = File.read(io_path)
     io = JSON.parse(io_c, symbolize_names: true)
-    if JSON::Validator.validate!(schema, io) # process JSON entries
-      # Clear any stored log messages ... TO DO
 
-      if io.has_key?(:psis) # library of linear thermal bridges
-        io[:psis].each do |p| psi.append(p); end
-      end
+    # schema validation is not yet supported in the OpenStudio Application
+    if schema_path
+      require "json-schema"
```

Comment:
@rd2 it would be difficult to include the `json-schema` gem with the measure.  Instead, I would suggest making the measure skip schema validation.  You can point users to an online JSON schema validation tool to check their JSON files after they have modified them

Reply:
Noted.